roles/content_view_publish/meta/main.yml

```diff
@@ -0,0 +1,21 @@
+galaxy_info:
```

Comment:
we usually don't add this file, as those roles are not to be uploaded to galaxy individually